app/models/wpcc/contribution_calculator.rb

```diff
@@ -0,0 +1,38 @@
+require 'active_model'
+
+module Wpcc
+  class ContributionCalculator
+    include ActiveModel::Model
+
+    UPPER_EARNINGS_THRESHOLD = 45_000
+    LOWER_EARNINGS_THRESHOLD = 5_876
```

Comment:
Are this numbers could be changed in every financial year? Can this number lives in a yaml file?